### ISSUE TITLE
List and double-check the checksums, please.

### ISSUE THREAD

**Issue by Erquint** [RobertTheSable/fe3-translation#1]
Found through the Romhacking.net, where you list checksums.
You rather ambiguously list separate "File" checksums alongside "ROM" checksums.
Please do also clarify the distinction as snes9x gives me the exact same checksum in its rom infodump as I calculate from the file on disk.
The two files I tried matched the "ROM" checksums for 1.0 and 1.1 you list.
However, after trying numerous combinations of source files and patches you provide, not a single resulting file matches any of the expected resulting checksums.
I'm using Lunar IPS, which worked just fine just recently for other roms.
I suspect that the checksums listed on Romhacking.net are erroneous. (Possibly out of date.)
And there's still the confusion in regards to your distinction of "File" vs "ROM".
Please, introduce some much-needed clarity by listing valid checksums in the README.

**Comment by RobertTheSable** [comment]
The file vs ROM thing is RHDN's doing, not mine. It's what their hasher gives you if you scan a headered rom.

Here's their autogenerated output for scanning a headered 1.0 rom:
```
Database match: Fire Emblem - Monshou no Nazo (Japan)
Database: No-Intro: Super Nintendo Entertainment System (v. 20210222-050638)
File SHA-1: B1C82729077A48993621DC1D294A7759557A90CD
File CRC32: 23B7227
ROM SHA-1: 3EE8CBE6638DA64F1D91DF9B81B1478498A5999F
ROM CRC32: 25D214F4
```

Presumably the File checksums include the header and the ROM fields do not.

Is this a major issue? It's been 2 years since I last updated the checksums and I haven't seen any issues from anyone. If you apply the right patch and the game runs, there's nothing too worry about.

RHDN only requires checksums for unmodified roms. And those haven't changed.

**Comment by RobertTheSable** [comment]
If it's really a concern, here are md5's for the most recent patch:
```
3a6bc7804fe8842ebdebfd1ed6e1fc09  v1.0_headered.smc
4d6ea7343a552ee17557861d861262d5  v1.0_unheadered.sfc
ef044f99ffd16c2ae8f981a19ad57570  v1.1_headered.smc
eb2c31971709221382590caa278f6052  v1.1_unheadered.sfc
```

**Comment by Erquint** [comment]
Thank you, this is rather helpful!

Patched a headerless 1.1 rom, the resulting checksum matches and it finally runs well.

For sake of posterity, if anyone falls down the same rabbit hole as me, here are more checksums for patched headerless rom files:
(Your headers may vary, resulting in wildly different hashing, so best strip them before patching for uniformity.)
```
Unpatched:

1.0

CRC32:  25D214F4
CRC64:  D47F4D8BF8842159
MD4:    A611B5C8DFCBDA271781439FF6AA5038
MD5:    2D674A5B82F7691CDB27702031D0F960
SHA1:   3EE8CBE6638DA64F1D91DF9B81B1478498A5999F
SHA256: 23E45B26DC5A73D88173C5CB7FA7FAE682EAB6E55D657A71B3B366DC95D603CA

1.1

CRC32:  A427B7E6
CRC64:  D8A4F932D0E2ADF8
MD4:    EC1D309529DAE7FEBD9BD7CC5C7C50CD
MD5:    0CB432DD9D5C1AFA5048334B110FDDC0
SHA1:   B0B03831D9104E3C5DF551D50EBC275A1692149E
SHA256: CCD5D8E3415D08EE949297D78ADC05D68EC945C6CC1FF195F1975A47095A3F18

Patched:

1.0

CRC32:  E369D10E
CRC64:  62CBC49175CA51FD
MD4:    8FD57490684ED8B3C281ED56BE3647BF
MD5:    4D6EA7343A552EE17557861D861262D5
SHA1:   34FF9D724B005A422518C6B7CF40B37D5AF8B379
SHA256: A18D4AAD9A6A771B5C4DA85E415DCA070C991BE58AD3F03CAE1C9C770F780338

1.1

CRC32:  0D2FF282
CRC64:  A9580A2C28BE412A
MD4:    995501E4F52F0D6B5E9885FFA586685C
MD5:    EB2C31971709221382590CAA278F6052
SHA1:   C0200C357FED3E8B32A4A6A84DECC99722F853F7
SHA256: D9DB5D5CFBB13E387EBDA9199CB2E854AE02D6C27EAF2B295E5D3B0AA7B9023C
```

To clarify, the first couple of times I tried to apply the patch resulted in files that didn't match the checksums and while one would not play at all, the other would be all jumbled up and unplayable as well.
That's when I decided to just run through all combinations until one matches the resulting checksum listed on Romhacking.net, since that would ultimately determine that everything worked as intended.
However, none of the resulting files matched the checksums there and the ones I tried running wouldn't play either.
Really seems now like it's out of date and perhaps in need of editing.

Another point of confusion was that this line from your readme…
> No 1.1 transition patch was included because the Quirino 1.1 patch converts the ROM to 1.0.

…coupled with there only being a **single** resulting checksum listed on Romhacking.net, gave me a false speculative impression that no matter which route I take — I should be arriving at the same exact file and the 1.1 revision changes somehow aren't relevant anymore. (I take partial blame for an overreaching interpretation.)


Thank you for your work on the patch and your assistance with checksums!
